docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -36,7 +36,7 @@ Server? The strategy that governs this behaviour is the
 * `{label}` which is a server side feature labelling a "versioned" set of config files.
 
 Repository implementations generally behave just like a Spring Boot
-application loading configuration files from a "spring.config.name"
+application loading configuration files from a "spring.application.name"
```

Comment:
I think this is actually `spring.config.name` (see [here](https://github.com/spring-cloud/spring-cloud-config/blob/master/spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/NativeEnvironmentRepository.java#L178)).  It means find `<appname>.{yml|properties}`, rather than `application.{yml|properties} (which applies to all apps).